src/classes/CRLP_RollupUI_TEST.cls

```diff
@@ -105,14 +105,42 @@ public class CRLP_RollupUI_TEST {
         System.assertNotEquals(null, jsonModel, 'JSON string of entire model should be returned.');
     }
 
+    /***
+    * @description Verifies if there are no exception thrown in setting up the Rollups when Einstein
+    * insights platform is enabled in the org
+    */
+    @isTest
+    private static void shouldNotThrowExceptionInCRLPSetupWhenEinsteinIsEnabled() {
```

Comment:
What would you think about a test method that validates the non-returned values from the `CRLP_RollupUI_SVC.getDetailFieldsWithType()` method:
- Call the method with the Account object
- Validate that the response does not include the `npe01__SYSTEM_AccountType__c` field (for example)